app/services/ncr/work_order_value_normalizer.rb

```diff
@@ -41,6 +42,12 @@ def normalize_soc_code
       end
     end
 
+    def normalize_approving_official_email
+      if !work_order.approving_official_email.present? && work_order.approvers.any?
+        work_order.approving_official_email = work_order.approvers.first.try(:email_address)
```

Comment:
maybe we should create a tech debt card for investigating this, then? feels icky